app/src/main/java/com/sapuseven/untis/activities/SettingsActivity.kt

```diff
@@ -242,33 +243,76 @@ class SettingsActivity : BaseActivity(), PreferenceFragmentCompat.OnPreferenceSt
 						}
 					}
 					"preferences_info" -> {
-						findPreference<Preference>("preference_info_app_version")?.summary = let {
-							val pInfo = requireContext().packageManager.getPackageInfo(requireContext().packageName, 0)
-							requireContext().getString(R.string.preference_info_app_version_desc,
-									pInfo.versionName,
-									PackageInfoCompat.getLongVersionCode(pInfo)
+						findPreference<Preference>("preference_info_app_version")?.apply {
+							val pInfo =
+								requireContext().packageManager.getPackageInfo(
+									requireContext().packageName,
+									0
+								)
+							summary = requireContext().getString(
+								R.string.preference_info_app_version_desc,
+								pInfo.versionName,
+								PackageInfoCompat.getLongVersionCode(pInfo)
 							)
+							setOnPreferenceClickListener {
+								startActivity(
+									Intent(
+										Intent.ACTION_VIEW,
+										Uri.parse("$REPOSITORY_URL_GITHUB/releases")
+									)
+								)
+								true
+							}
 						}
-
 						findPreference<Preference>("preference_info_github")?.apply {
 							summary = REPOSITORY_URL_GITHUB
 							setOnPreferenceClickListener {
-								startActivity(Intent(Intent.ACTION_VIEW, Uri.parse(REPOSITORY_URL_GITHUB)))
+								startActivity(
+									Intent(
+										Intent.ACTION_VIEW,
+										Uri.parse(REPOSITORY_URL_GITHUB)
+									)
+								)
 								true
 							}
 						}
+						findPreference<Preference>("preference_info_license")?.setOnPreferenceClickListener {
+							startActivity(
+								Intent(
+									Intent.ACTION_VIEW,
+									Uri.parse("$REPOSITORY_URL_GITHUB/blob/master/LICENSE")
+								)
+							)
+							true
+						}
 					}
 					"preferences_contributors" -> {
-						GlobalScope.launch(Dispatchers.Main) {
-							"https://api.github.com/repos/sapuseven/betteruntis/contributors"
-									.httpGet()
-									.awaitStringResult()
-									.fold({ data ->
-										showContributorList(true, data)
-									}, {
-										showContributorList(false)
-									})
-						}
+						AlertDialog.Builder(requireContext())
```

Comment:
Use `MaterialAlertDialogBuilder` instead
```suggestion
						MaterialAlertDialogBuilder(requireContext())
```